scripts/service_alerts/add_service_alert.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+from google.cloud import storage as gcs
+import json
+from utils import *
```

Comment:
minor: generally prefer to explicitly name all imports rather than use `import *`

Reply:
Updated in 0e41967a1dcf755f16042f8e5c1ec94e40125c51